lib/platform/gitlab/types.ts

```diff
@@ -8,6 +8,35 @@ export interface GitlabComment {
   id: number;
 }
 
+export type GitLabUser = {
```

Comment:
Use type or interface for all, do not mix 🤗

Reply:
My thought was that I would use `type` for all types and `interface` for all function parameters, but tbh I've always thought that TS doesn't really make a good enough distinction between types and interfaces so in the past I've always defaulted to just using interfaces everywhere :man_shrugging: .
I guess based on the TS docs: `If you would like a heuristic, use interface until you need to use features from type.`, I'll change these to interfaces.